packages/discovery-react-components/src/components/SearchFacets/utils/searchFilterTransform.tsx

```diff
@@ -93,10 +93,33 @@ export class SearchFilterTransform {
     const filterStrings: string[] = [];
     facets.forEach(facet => {
       const field = get(facet, 'field', '');
-      const results = get(facet, 'results', []);
-      const keys = this.quoteSelectedFacets(results, 'key');
-      if (keys.length) {
-        filterStrings.push(keys.map(key => `${escapeFieldName(field)}::${key}`).join('|'));
+      const results: SelectableQueryTermAggregationResult[] | never[] = get(facet, 'results', []);
+      const hasCategories = // logic duplicated from CollapsibleFacetsGroup
+        field.includes('enriched_') &&
+        field.includes('entities.text') &&
+        results?.[0]?.aggregations !== undefined;
+      // if a group of facets contain sub-categories, surround each sub-category with an AND (,)
+      if (hasCategories) {
+        const resultsByCategory: { [key: string]: SelectableQueryTermAggregationResult[] } = {};
```

Comment:
Nit: I've been finding that using `Record<>` makes the code a bit more readable:
```suggestion
        const resultsByCategory: { Record<string, SelectableQueryTermAggregationResult[]> } = {};
```